aries-site/src/pages/components/accordion.js

```diff
@@ -18,23 +19,72 @@ const Accordion = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
-        <SubsectionText>
-          The accordion affords content to be delivered progressively. The
-          chevron is used to identify the expand or collapse action while the
-          entire header can be clicked to expand or collapse content.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
```

Comment:
Looks like Accordion description needs to be updated in structure. It's pulling in the Button info